datasource/src/test/scala/quasar/physical/mongo/MongoSpec.scala

```diff
@@ -179,27 +187,57 @@ object MongoSpec {
   val BatchSize: Int = 64
 
   def mkMongo: IO[Disposable[IO, Mongo[IO]]] =
-    Mongo[IO](MongoConfig(connectionString, BatchSize, PushdownLevel.Full))
+    Mongo[IO](MongoConfig(connectionString, BatchSize, PushdownLevel.Full, None))
+
+  def privateKey: IO[String] = IO.delay {
+    val path = Paths.get("key_for_docker")
+    new String(Files.readAllBytes(path))
+  }
+
+  val TunneledURL: String = "mongodb://root:secret@mng:27017"
+  val TunnelUser: String = "root"
+  val TunnelPassword: String = "root"
+  val TunnelPassphrase: Option[String] = Some("passphrase")
+  val TunnelHost: String = "localhost"
+  val TunnelPort: Int = 22222
```

Comment:
We should consider starting to split up the tests into unit tests (which shouldn't require any of the external process dependencies, and thus don't touch SSH or mongo directly), and integration tests (which… do).

Reply:
Currently `MongoSpec` literally just checks that connection could be established and works directly with different connection strings. Other tests depend on running mongo, but don't check if it's OK, just checking aggregation, config codecs etc. What is desired behaviour here?

Reply:
Could be useful to reorganize the tests a bit indeed, but maybe in another PR? 

Running the full test suite also takes quite a long time. We can then also look into making that less time consuming.